packages/experimental/src/components/AboutModal/AboutModal.js

```diff
@@ -109,14 +104,10 @@ export const AboutModal = ({
             </ModalBody>
             <ModalFooter>
               {technologiesUsed && technologiesUsed.length ? (
-                <Tabs
-                  className={`${pkgPrefix}-about-modal-tab-container`}
-                  light={theme === 'light'}
-                  aria-label="About modal technology used and version number tabs">
+                <Tabs className={`${pkgPrefix}-about-modal-tab-container`}>
                   <Tab
                     id="about-modal-technologies-used-tab"
-                    label="Technologies used"
-                    aria-label="Technologies used tab">
+                    label="Technologies used">
```

Comment:
Strings should be a prop for translation purposes

Reply:
That's correct, they should: this will be dealt with for issue #383 which will restructure the way these tabs are supplied in a way that enables the necessary text to be passed in as props. This PR just addresses part of issue #389.